docs/content/how-to/agent/create-agent-cluster.md

```diff
@@ -416,7 +416,7 @@ In order to get the cluster in a running state we need to add some nodes to it.
 We add nodes to our HostedCluster by scaling the NodePool object. In this case we will start by scaling the NodePool object to two nodes:
 
 ~~~sh
-oc -n ${CLUSTERS_NAMESPACE} scale nodepool ${HOSTED_CLUSTER_NAME} --replicas 2
+oc -n ${CLUSTERS_NAMESPACE} scale nodepool ${NODEPOOL_NAME} --replicas 2
 ~~~
 
 The ClusterAPI Agent provider will pick two agents randomly that will get assigned to the HostedCluster. These agents will go over different states and will finally join the HostedCluster as OpenShift nodes.
```

Comment:
It looks like the main update here is to a technical element, so I'll defer to the other reviewers as to whether the change is accurate. If you're interested in updating this text to better align with the style and grammar that we use in the OCP docs, you can update the text as follows:

```
To run the cluster, you need to add nodes to it by scaling the NodePool object. Scale the NodePool object to two nodes:
~~~sh
oc -n ${CLUSTERS_NAMESPACE} scale nodepool ${NODEPOOL_NAME} --replicas 2
~~~

The ClusterAPI Agent provider randomly picks two agents that are assigned to the hosted cluster. Those agents go through different states and finally join the hosted cluster as OpenShift Container Platform nodes.
```